v5/getting-started/macos.rst

```diff
@@ -13,7 +13,8 @@ The recommended method of installing PROS 3 for macOS involves using `Homebrew <
 2. Once you have Homebrew installed, run :code:`brew tap osx-cross/arm && brew install arm-gcc-bin` to register a repository with Homebrew that contains the toolchain used to build PROS projects, and then install the toolchain.
 3. Run :code:`brew tap purduesigbots/pros` to register the PROS Homebrew repository with Homebrew.
 4. Run :code:`brew cask install pros-editor` to install the PROS Editor (the CLI will also be installed). This may also take a while.
-5. That's it! You can now start using PROS 3.
+5. (Optional) If you are planning to use the Vision Sensor, you will likely need to also install the VEX Vision Utility to set up signatures. Run :code:`brew install vcs-vision` to do this. Once installed, you'll be able to run the program by looking for "vcs_vision" in Spotlight.

```

Comment:
is the search term in spotlight really vcs_vision? i thought it shows up as a normal program (like "VCS Vision") also

Reply:
Yes the search term is really vcs_vision
<img width="616" alt="Screen Shot 2020-12-21 at 5 47 34 PM" src="https://user-images.githubusercontent.com/22580992/102829216-9cdf8280-43b4-11eb-93a0-fce8590c8e07.png">